### PR TITLE
[SMALL] adding parameterless ctor to SqlFunctionExpression

### PR DESCRIPTION
**Pull request by maumar**

### CODE REVIEW

**Comment by maumar** [comment]
@anpete

File: src/EntityFramework.Relational/Query/Expressions/SqlFunctionExpression.cs [hunk 1]
```diff
@@ -20,12 +20,19 @@ public class SqlFunctionExpression : Expression
...
+
+        public SqlFunctionExpression(
+            [NotNull] string functionName,
+            [NotNull] Type returnType,
+            [NotNull] IEnumerable<Expression> arguments)
         {
             FunctionName = functionName;
-            _arguments = new ReadOnlyCollection<Expression>(arguments.ToList());
             Type = returnType;
+            _arguments = new ReadOnlyCollection<Expression>(arguments.ToList());
```
Comment (anpete) [inline src/EntityFramework.Relational/Query/Expressions/SqlFunctionExpression.cs]: arguments.ToList().AsReadOnly()

**Comment by anpete** [comment]
:shipit:

**Comment by natemcmaster** [comment]
:+1: I need this in #3011

**Comment by maumar** [comment]
merged b5de4952b5a459bcd7ad5d001a4ac9b4d2cb6508

**Pull request closed without merging**